_articles/dual-booting.md

```diff
@@ -78,10 +41,12 @@ This message can be safely ignored and it is caused by Windows not being the fir
 
 ## Using another drive
 
-Another way to set up a dual boot is to install another drive for the other OS of your choice. This is one of the easiest ways to dual boot as each OS will set up the whole drive for automatically created partitions and won't require you to resize any partitions. To access each OS you would reboot and hold the boot menu key (F7 for our laptops and F12/Del for our desktops).
+Another way to set up a dual boot is to install another drive for the other OS of your choice. This is one of the easiest ways to dual boot as each OS will set up the whole drive for automatically created partitions and won't require you to resize any partitions. To access each OS you would reboot and hold the boot menu key (F7 for our laptops and F10/F12/Del for our desktops).
 
 ## Windows Caveats
 
 Windows 8 and later uses a "Fast Startup" setting which prevents Windows from fully shutting down and allowing other OSes to use the disk. Before you can properly dual boot with Windows, you must disable this setting in Windows.
 
 In your Windows install, open Control Panel and head to "Power Options" Select "Choose what the power buttons do", select "Change settings that are currently unavailable", then disable the "fast startup" setting. Note that Windows updates may occasionally turn this setting back on without asking, so if you are unable to boot into Pop!\_OS, check this setting first.
+
```

Comment:
SUGGESTED ADDITION

If you want to take additional steps to prevent "fast startup" causing any issues, you can turn off automatic updates for Windows [Instructions/Link]. However, if you choose to do so, it is recommended that you manually check for and apply Windows updates, especially security updates daily/weekly, then check to make sure that "fast startup" is still turned off before rebooting.